examples/read_nodes_serde.rs

```diff
@@ -0,0 +1,81 @@
+// note: to use serde, the feature needs to be enabled
+// run example with:
+//    cargo run --example read_nodes_serde --features="serialize"
+
+use quick_xml::de::from_str;
+use serde::Deserialize;
+
+#[derive(Debug, PartialEq, Default, Deserialize)]
+#[serde(default)]
+struct Translation {
+    #[serde(rename = "@Tag")]
+    tag: String,
+    #[serde(rename = "@Language")]
+    lang: String,
+    #[serde(rename = "Text")]
+    text: String,
+}
+
+#[derive(Debug, PartialEq, Default, Deserialize)]
+#[serde(default)]
+struct DefaultSettings {
+    #[serde(rename = "@Language")]
+    language: String,
+    #[serde(rename = "@Greeting")]
+    greeting: String,
+}
+
+#[derive(Debug, PartialEq, Default, Deserialize)]
+#[serde(default)]
+struct Config {
+    #[serde(rename = "DefaultSettings")]
+    settings: DefaultSettings,
+    #[serde(rename = "Localization")]
+    translation: Vec<Translation>,
+}
```

Comment:
[This should be](https://docs.rs/quick-xml/latest/quick_xml/de/index.html#element-lists):
```suggestion
#[derive(Debug, PartialEq, Default, Deserialize)]
#[serde(rename_all = "CamelCase")]
struct Config {
    #[serde(rename = "DefaultSettings")]
    settings: DefaultSettings,
    localization: Localization,
}
#[derive(Debug, PartialEq, Default, Deserialize)]
#[serde(rename_all = "CamelCase", default)]
struct Localization {
    translation: Vec<Translation>,
}
```
I think, it is also would better to use `#[serde(rename_all = "...")]` attribute whenever possible and keep individual `#[serde(rename = "...")]` only where necessary. This makes the code cleaner at my glance.